wdl-engine/CHANGELOG.md

```diff
@@ -9,36 +9,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Added
 
-* Implemented remote file localization for task execution (#[386](https://github.com/stjude-rust-labs/wdl/pull/386)).
-* Implemented concurrent file downloads for localization for task execution (#[424](https://github.com/stjude-rust-labs/wdl/pull/424)).
+* Implemented remote file localization for task execution ([#386](https://github.com/stjude-rust-labs/wdl/pull/386)).
+* Implemented concurrent file downloads for localization for task execution ([#424](https://github.com/stjude-rust-labs/wdl/pull/424)).
 
 #### Fixed
 
 * Fix overly verbose call stacks in task failure messages ([#435](https://github.com/stjude-rust-labs/wdl/pull/435))
-* Fix `sub` replacement of multiple instances (#[426](https://github.com/stjude-rust-labs/wdl/pull/426)).
-* Fix path translation in more expressions (#[422](https://github.com/stjude-rust-labs/wdl/pull/422)).
-* The `sep` placeholder option was not performing guest path translation (#[417](https://github.com/stjude-rust-labs/wdl/pull/417)).
+* Fix `sub` replacement of multiple instances ([#426](https://github.com/stjude-rust-labs/wdl/pull/426)).
+* Fix path translation in more expressions ([#422](https://github.com/stjude-rust-labs/wdl/pull/422)).
+* The `sep` placeholder option was not performing guest path translation ([#417](https://github.com/stjude-rust-labs/wdl/pull/417)).
 * Placeholder options are now type checked at runtime ([#345](https://github.com/stjude-rust-labs/wdl/pull/345)).
 * Whether or not a task manager state represents unlimited resources is now correctly calculated ([#397](https://github.com/stjude-rust-labs/wdl/pull/397)).
 * Fixed environment variable values are not using guest paths for Docker backend ([#398](https://github.com/stjude-rust-labs/wdl/pull/398)).
 * Ensure output files created by Docker tasks running as root have correct host user permissions ([#379](https://github.com/stjude-rust-labs/wdl/pull/379)).
-* Fixes `chown` functionlity by making the path absolute ([#428](https://github.com/stjude-rust-labs/wdl/pull/379)).
+* Fixes `chown` functionality by making the path absolute ([#428](https://github.com/stjude-rust-labs/wdl/pull/379)).
 
 #### Changed
 
-* Evaluation errors now contain a "backtrace" containing call locations (#[432](https://github.com/stjude-rust-labs/wdl/pull/432)).
-* Changed origin path resolution in inputs to accomodate incremental command line parsing ([#430](https://github.com/stjude-rust-labs/wdl/pull/430)).
+* Refactored `crankshaft` backend to the `docker` backend ()
```

Comment:
needs the link

Reply:
😆 I got distracted by other defects in the file I forgot to paste it. Fixing.